docs/journalist.rst

```diff
@@ -1,289 +1,299 @@
 Journalist Guide
 ================
 
-Journalists viewing documents on SecureDrop must connect to the
-respective Source or Journalist Interface using the `Tails operating
-system <https://tails.boum.org/>`__, which your administrator should
-have already set up for you.
-
-Workflow
---------
+This guide presents an overview of the SecureDrop system for a
+journalist. It covers the core functions necessary to start working
+with the platform: logging in securely, viewing documents, editing
+documents, and interacting with sources.
 
-.. todo:: This whole document needs to be reworked, it's all over the
-          place and does not flow from top to bottom.
+Creating a GPG Key
+------------------
 
-Create GPG key for the journalist
----------------------------------
+Each journalist needs a personal GPG key for encrypting files. A GPG
+key has two parts: a *public key* and a *private key*. The private
+key, used for decryption, stays on the *Journalist Workstation*. The
+public key, used for encryption, is copied to the *Secure Viewing
+Station*.
 
-Each journalist must have a personal GPG key that they use for
-encrypting files transferred from the *Secure Viewing Station* to their
-Journalist Workstation. The private key, used for decryption, stays on
-their Journalist Workstation. The public key, used for encryption, gets
-copied to the *Secure Viewing Station*.
+.. todo:: This document recommends transferring documents from the
+          *SVS* to the *Journalist Workstation*, without any
+          discussion of the potential risks or mitigations that should
+          be taken when doing so. A section needs to be added on why
+          doing this could be risky, and what can be done to make this
+          situation better.
 
-.. todo:: This document recommends transferring documents from the *SVS*
-          to the Journalist Workstation, without any discussion of the
-          potential risks or mitigations that should be taken when
-          doing so. A section needs to be added on why doing this
-          could be risky, and what can be done to make this situation
-          better.
+If you do not yet have a GPG key, follow the instructions for your
+operating system to set one up with GnuPG:
 
-If a journalist does not yet have a GPG key, they can follow these
-instructions to set one up with GnuPG (GPG).
+- `GNU/Linux <https://www.gnupg.org/gph/en/manual.html#AEN26>`__
+- `Windows <http://gpg4win.org/>`__
+- `Mac OS X <https://support.gpgtools.org/kb/how-to/first-steps-where-do-i-start-where-do-i-begin>`__
```

Comment:
Huh, these are not really guides. We should link to https://ssd.eff.org guides, specifically:

* GNU/Linux: https://ssd.eff.org/en/module/how-use-pgp-linux
* Mac OS: https://ssd.eff.org/en/module/how-use-pgp-mac-os-x
* Windows: https://ssd.eff.org/en/module/how-use-pgp-windows

Note: this is not required for merge, just noting this while I read through this document and noting here for the future.